ui/src/layouts/chats/presentation/chat/edit_group.rs

```diff
@@ -44,6 +45,16 @@ pub fn EditGroup(cx: Scope) -> Element {
     // show the ADD or REMOVE components, default to Remove
     let edit_group_action = use_state(cx, || EditGroupAction::Remove);
     let conv_id = state.read().get_active_chat().unwrap().id;
+    let creator_id_vector = Vec::from_iter(
+        state
+            .read()
+            .get_active_chat()
+            .unwrap()
+            .creator
+            .iter()
+            .cloned(),
+    );
+    let creator_id = creator_id_vector.first().cloned()?;
```

Comment:
You can try the creator field (and might as well do the same for `get_active_chat()` unless you wish to panic to treat it as a bug if its `None`?)

```suggestion
    let creator_id = state.read().get_active_chat()?.creator.clone()?;
```

Reply:
Done. Thanks for the tip!